crates/aptos/src/account/create_resource_account.rs

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::common::types::{CliCommand, CliTypedResult, TransactionOptions};
+use aptos_rest_client::{
+    aptos_api_types::{WriteResource, WriteSetChange},
+    Transaction,
+};
+use aptos_transaction_builder::aptos_stdlib;
+use aptos_types::account_address::AccountAddress;
+use async_trait::async_trait;
+use clap::Parser;
+use serde::Serialize;
+
+/// Command to create a resource account
+///
+#[derive(Debug, Parser)]
+pub struct CreateResourceAccount {
+    #[clap(flatten)]
+    pub(crate) txn_options: TransactionOptions,
+
+    /// Resource account seed.
+    #[clap(long)]
+    pub(crate) seed: String,
+
+    /// Optional Resource Account authentication key.
+    #[clap(long, parse(try_from_str = crate::common::types::load_account_arg))]
+    pub(crate) authentication_key: Option<AccountAddress>,
```

Comment:
https://github.com/aptos-labs/aptos-core/blob/main/types/src/transaction/authenticator.rs#L365
why not use AuthenticationKey? There is a from string operator. And while these may be the same under the hood, there is a semantic difference.

Reply:
I have switched to use the `AuthenticationKey` as suggested.